src/Microsoft.Health.Dicom.Functions/Configuration/DicomFunctionsConfiguration.cs

```diff
@@ -0,0 +1,20 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Health.Dicom.Functions.Management;
+
+namespace Microsoft.Health.Dicom.Functions.Configuration
+{
+    internal static class DicomFunctionsConfiguration
+    {
+        public const string SectionName = "DicomFunctions";
+
+        public const string PurgeFrequencyVariable = "%"
```

Comment:
nit: I'm thinking should we move this constant into the job class.
So far the `xxConfiguration` includes 2 things: `SectionName` and configurations. This one is pretty special.

Reply:
Moved